ru/turing/courses/lesson2/animals/Animal.java

```diff
@@ -1,4 +1,31 @@
 package ru.turing.courses.lesson2.animals;
 
-public class Animal {
+public abstract class Animal {
+    // статические поля
+    protected static int hungerRate;
+
+    // динамические поля
+    protected int age;
+    protected int hunger;
+    protected String name;
+
+    public void Animal(int age, int hunger, String name) {
+        if(name ==null)
+            return;
+        this.age = age;
+        this.hunger = hunger;
+        this.name = name;
+    }
+
+    public void Eat(int nutrition) {
```

Comment:
```suggestion
    public void eat(int nutrition) {
```
С заглавной пишем только названия классов